ops/Cargo.toml

```diff
@@ -36,3 +36,6 @@ spl-account-compression = { workspace = true }
 sqlx = { workspace = true }
 thiserror = { workspace = true }
 tokio = { workspace = true }
+mpl-token-metadata = { workspace = true }
+serde_json = { workspace = true }
+txn_forwarder = { workspace = true }
```

Comment:
I want to delete the tools directory once ops cli covers all the use cases. There is rpc client available in core which does retries so please look to use that instead.